tsconfig.base.json

```diff
@@ -15,6 +15,9 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@daohaus-monorepo/tx-builder-feature": [
```

Comment:
I'm curious why it generated w/ @daohaus-monorepo instead of @daohaus

Reply:
and i see it the other way here:

https://github.com/HausDAO/daohaus-monorepo/pull/304/files#diff-520b36df452487d613b8ea62dfc687f081a918373e6e085bebf388136f0d6e0aR2

feel like we need to fix this before merging. can manually change it.